src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -202,6 +216,8 @@ func DeployCmd(noMetrics *bool) *cobra.Command {
 		"Deploying on AWS with cloudfront is currently "+
 			"an alpha feature. No security restrictions have been"+
 			"applied to cloudfront, making all data public (obscured but not secured)")
+	deployAmazon.Flags().StringVar(&creds, "credentials", "", "Use the format '--credentials=<id>,<secret>,<token>'\n<id>, <secret>, and <token> are session token details, used for authorization. You can get these by running 'aws sts get-session-token'")
```

Comment:
It seems a little bit weird to pack multiple string arguments into a single one, although we do that in other places like with commits that's normally for a list of things that are the same type. How would you feel about having separate `--id`, `--secret`, `--token` flags? That strikes me as a little more standard for command line interfaces.

Reply:
I feel like having three flags is kinda painful.  Personally speaking I prefer a single comma-separated `--credentials` flag like the way it is.  I guess if I had to give a reason, it'd be that it wouldn't make sense to provide, say, an `--id` flag without a `--secret`, though `--token` is admittedly optional.

Reply:
I don't feel to strongly so this will work. Are all the places we use the old scheme changed in the docs for this pr?

Reply:
Yup.